storybookjs/storybook
# Addon essentials

Issue: #5731

## What I did

- [x] Add a preset to `addon-knobs` that automatically installs its decorator (unless disabled)
- [x] Add `addon-essentials`, a curated set of addons:
  - w/o presets: `actions`, `backgrounds`, `links`, `viewport`
  - w/ presets: `knobs`, `docs` (to be removed before release)
- [x] Add a sample app `cra-ts-essentials` that demonstrates out-of-box behavior
- [x] Warn to let users know that there may be overlapping configuration if any of  its addons are detected in their app

## How to test

See the `addon-essentials` README for how to configure, and `cra-ts-essentials` for a test environment.
